Makefile

```diff
@@ -26,4 +28,15 @@ tests: sandbox
 	cabal test
 	./tests/test_binary.sh
 
+test-in-docker: Dockerfile.build
+	sudo docker network create welder
+	# download metadata and run the API backend which provides depsolving
+	[ -f "metadata.db" ] || curl https://s3.amazonaws.com/weldr/metadata.db > metadata.db
+	sudo docker ps | grep api || sudo docker run -d --rm --name api -p 4000:4000 -v `pwd`:/mddb --security-opt label=disable --network welder welder/bdcs-api-rs:latest
+
+	sudo docker build -t $(ORG_NAME)/bdcs-cli:latest -f $< --cache-from $(ORG_NAME)/bdcs-cli:latest .
```

Comment:
I think we need to figure out how to make the --cache-from conditional. We should be able to run these on our local dev systems with the Fedora docker packages.

Reply:
Agree. How about I deal with this separately b/c the --cache-from option is used at several different places in the welder repos ?

Reply:
PS: I can try aliasing `docker build` to something which passes the `--cache-from` option if we are in CI but it's not going to be very pretty.

Reply:
Yeah, a separate set of patches is fine.